src/App.js

```diff
@@ -0,0 +1,34 @@
+import './styles/App.scss';
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import MainPage from './components/HomePage/MainPage';
+import Details from './components/DoctorsData/DetailsPage/Details';
+import Appointments from './components/AppointmentsData/AppointmentsPage';
+import LoginPage from './components/Authentication/LoginPage';
+import RegistrionPage from './components/Authentication/RegistrionPage';
+import SplashPage from './components/SplashPage/SplashPage';
+import AddItem from './components/DoctorsData/AddItem';
+import DeleteItem from './components/DoctorsData/DeleteItem';
+import AppointmentsDetails from './components/AppointmentsData/DetailsPage/Details';
+import AddAppointmentsForm from './components/AppointmentsData/AddAppointment';
+
+function App() {
```

Comment:
- [ ] Kindly, remove the old pieces of commented code present in:

- AddAppointment.js
- LoginPage.jsx
- Navbar.js

If you know there is more commented code on your created files that I couldn't find, please remove it.

That way, the cleanliness and tidiness of the repo will be improved. 👍